src/main/resources/changelog.xml

```diff
@@ -18,6 +18,20 @@
                 <constraints nullable="false" unique="false"/>
             </column>
         </createTable>
+        <createTable tableName="caretakers">
```

Comment:
This is wrong. You have already added a change set number 2, but you are changing change set number 1. So if you did this in production, you would get an error, because the change sets would differ. So for adding a new table, you have to add a new change set :)

Reply:
I think I haven't understood your point.
I will check it with you on Monday.